notebooks/cytomining-ecosystem/visualize-landscape-fall-2023.py

```diff
@@ -0,0 +1,688 @@
+        Many tools from GitHub queries show a lack of updates for greater than 2 years.
+        """,
+        "section": "Maturity",
+    }
+)
+
+fig_maturity_latest_commit.show()
+
+# +
+# scatter plot for network and subscribers
+
+# add log of github stars to help visualize
+df_projects["GitHub Forks (Log Scale)"] = np.log(
+    df_projects["GitHub Forks"].apply(
+        # move 0's to None to avoid divide by 0
+        lambda x: x
+        if x > 0
+        else None
+    )
+)
+
+fig_network_and_subscribers = px.scatter(
+    df_projects.sort_values(by="category"),
+    hover_name="Project Name",
+    x="GitHub Forks (Log Scale)",
+    y="GitHub Subscribers",
+    width=1200,
+    height=500,
+    color="category",
+    color_discrete_sequence=category_color_sequence,
+)
+
+
+# customize the chart layout
+fig_network_and_subscribers.update_layout(
+    title=f"Project Watchers and GitHub Forks (click categories for further focus)",
+    xaxis_title="GitHub Forks (Log Scale)",
+    yaxis_title="GitHub Watchers",
+)
+
+fig_collection.append(
+    {
+        "plot": fig_network_and_subscribers,
+        "description": """
+        This plot shows project GitHub watcher and GitHub fork count
+        as a way to demonstrate relative usage.
+        """,
+        "findings": """
+        We can see a wide variety of project watcher and fork count depending on the category involved.
+        Generally, projects of <code>loi-focus</code> are at an earlier stage and thus have fewer forks and watchers.
+        """,
+        "section": "Usage",
+    }
+)
+
+fig_network_and_subscribers.show()
+
+# +
+# scatter plot for contributors and issues
+
+# add log of github stars to help visualize
+df_projects["GitHub Open Issues (Log Scale)"] = np.log(
+    df_projects["GitHub Open Issues"].apply(
+        # move 0's to None to avoid divide by 0
+        lambda x: x
+        if x > 0
+        else None
+    )
+)
+
+
+fig_contributors_and_issues = px.scatter(
+    df_projects.sort_values(by="category"),
+    hover_name="Project Name",
+    x="GitHub Open Issues (Log Scale)",
+    y="GitHub Contributors",
+    width=1200,
+    height=500,
+    color="category",
+    color_discrete_sequence=category_color_sequence,
+)
+
+
+# customize the chart layout
+fig_contributors_and_issues.update_layout(
+    title=f"Project Open Issues and Contributors (click categories for further focus)",
+)
+
+fig_collection.append(
+    {
+        "plot": fig_contributors_and_issues,
+        "description": """
+        This plot shows the number of project contributors and issue counts as a way to describe usage through maintainability.
+        """,
+        "findings": """
+        We find that many projects have a high contributor and issue count.
+        A few Cytomining Ecosystem adjacent and microscopy analysis tools appear to be on a trend towards
+        relevant open source tools.
```

Comment:
```suggestion
        A few examples in the <code>Cytomining Ecosystem adjacent<code> and <code>microscopy analysis tool<code> categories are trending toward tools in the <code>relevant open source<code> category.
        relevant open source tools.
```

Reply:
I may have gotten one of the category names wrong

Reply:
Thanks - adding this in!